Example doesn't work

Hey, I am new for both Apache Beam and  DataFlow, and I've tried run `datasplash.examples`, but it raise an error message.

My command to run:
```bash
lein run -- word-count --input=gs://dataflow-samples/shakespeare/kinglear.txt --output=./results.txt
``` 
Error message:
```
Compiling 6 source files to /Users/xingxing/PXN/datasplash/target/classes
Exception in thread "main" Syntax error compiling at (/private/var/folders/dr/gn78q20n3kl8xmjz8_9kbz0r0000gn/T/form-init2152568073736720306.clj:1:125).
        at clojure.lang.Compiler.load(Compiler.java:7647)
        at clojure.lang.Compiler.loadFile(Compiler.java:7573)
        at clojure.main$load_script.invokeStatic(main.clj:452)
        at clojure.main$init_opt.invokeStatic(main.clj:454)
        at clojure.main$init_opt.invoke(main.clj:454)
        at clojure.main$initialize.invokeStatic(main.clj:485)
        at clojure.main$null_opt.invokeStatic(main.clj:519)
        at clojure.main$null_opt.invoke(main.clj:516)
        at clojure.main$main.invokeStatic(main.clj:598)
        at clojure.main$main.doInvoke(main.clj:561)
        at clojure.lang.RestFn.applyTo(RestFn.java:137)
        at clojure.lang.Var.applyTo(Var.java:705)
        at clojure.main.main(main.java:37)
Caused by: java.lang.IllegalArgumentException: .via() is required
        at org.apache.beam.repackaged.beam_sdks_java_core.com.google.common.base.Preconditions.checkArgument(Preconditions.java:122)
        at org.apache.beam.sdk.io.FileIO$Write.expand(FileIO.java:1212)
        at org.apache.beam.sdk.io.FileIO$Write.expand(FileIO.java:778)
        at org.apache.beam.sdk.Pipeline.applyInternal(Pipeline.java:537)
        at org.apache.beam.sdk.Pipeline.applyTransform(Pipeline.java:471)
        at org.apache.beam.sdk.values.PCollection.apply(PCollection.java:357)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at clojure.lang.Reflector.invokeMatchingMethod(Reflector.java:167)
        at clojure.lang.Reflector.invokeInstanceMethod(Reflector.java:102)
        at datasplash.core$tapply.invokeStatic(core.clj:432)
        at datasplash.core$tapply.invoke(core.clj:428)
        at datasplash.core$apply_transform.invokeStatic(core.clj:452)
        at datasplash.core$apply_transform.invoke(core.clj:445)
        at datasplash.core$write_text_file.invokeStatic(core.clj:1168)
        at datasplash.core$write_text_file.invoke(core.clj:1149)
        at datasplash.examples$run_word_count.invokeStatic(examples.clj:53)
        at datasplash.examples$run_word_count.invoke(examples.clj:45)
        at datasplash.examples$_main.invokeStatic(examples.clj:340)
        at datasplash.examples$_main.doInvoke(examples.clj:336)
        at clojure.lang.RestFn.invoke(RestFn.java:439)
        at clojure.lang.Var.invoke(Var.java:393)
        at user$eval140.invokeStatic(form-init2152568073736720306.clj:1)
        at user$eval140.invoke(form-init2152568073736720306.clj:1)
        at clojure.lang.Compiler.eval(Compiler.java:7176)
        at clojure.lang.Compiler.eval(Compiler.java:7166)
        at clojure.lang.Compiler.load(Compiler.java:7635)
        ... 12 more
```

--- a/src/clj/datasplash/core.clj
+++ b/src/clj/datasplash/core.clj
@@ -1113,15 +1113,18 @@ It means the template %A-%U-%T is equivalent to the default jobName"
 (def text-writer-schema
   {:file-format {:docstr "Choose file format."
                  :action
-                 (fn [transform file-format] (.via transform
-                                              (Contextful/fn
-                                                (sfn
-                                                 (fn [x]
-                                                   (case file-format
-                                                     :json (json/encode x {})
-                                                     :edn (pr-str x)
-                                                     (file-format x)))))
-                                              (TextIO/sink)))}
+                 (fn [transform file-format]
+                   (if (= :default file-format)
+                     (.via transform (TextIO/sink))
+                     (.via transform
+                           (Contextful/fn
+                             (sfn
+                              (fn [x]
+                                (case file-format
+                                  :json (json/encode x {})
+                                  :edn (pr-str x)
+                                  (file-format x)))))
+                           (TextIO/sink))))}
    :compression-type {:docstr "Choose compression type."
                       :enum compression-type-enum
                       :action (select-enum-option-fn
@@ -1158,13 +1161,14 @@ See https://cloud.google.com/dataflow/java-sdk/JavaDoc/com/google/cloud/dataflow
 ```"
           base-schema text-writer-schema)
    :added "0.6.2"}
-  [to {:keys [dynamic? dynamic-fn] :as options} ^PCollection pcoll]
-  (let [[base-path filename ] (split-path to)
+  [to {:keys [dynamic? dynamic-fn file-format] :as options} ^PCollection pcoll]
+  (let [[base-path filename] (split-path to)
         opts (-> options
-                  (assoc :label (str "write-text-file-to-"
-                                     (clean-filename to))
-                         :coder nil)
-                  (cond-> filename (assoc :prefix filename)))]
+                 (assoc :label (str "write-text-file-to-"
+                                    (clean-filename to))
+                        :coder nil
+                        :file-format (or file-format :default))
+                 (cond-> filename (assoc :prefix filename)))]
     (apply-transform pcoll
                      (-> (if dynamic?
                            (-> (FileIO/writeDynamic)

